src/govuk/components/character-count/template.njk

```diff
@@ -10,6 +10,7 @@
     name: params.name,
     describedBy: params.id + '-info',
     rows: params.rows,
+    spellcheck: params.spellcheck,
```

Comment:
Because the character count sort of 'extends' the textarea component rather than 'uses it', `spellcheck` needs to be added as a parameter in `character-count.yaml` as well (just like you added it to the textarea component):

https://github.com/alphagov/govuk-frontend/blob/ce8e22355e32af4d8f3c876779acd26128401842/src/govuk/components/textarea/textarea.yaml#L10-L13

Reply:
That seems correct - although I managed to test the change worked without this, which I guess makes me question what these params do...? Is it more documentation than anything which will fail if something isn't present?

Reply:
Great question!

These parameters are [included as a JSON file in the released package](https://github.com/alphagov/govuk-frontend/blob/ce8e22355e32af4d8f3c876779acd26128401842/package/govuk/components/textarea/macro-options.json) and then they're used to automatically generate the ['Nunjucks macro options' in the Design System](https://design-system.service.gov.uk/components/character-count/#options-character-count-example).

We've looked at trying to test that the params file is correct in the past – there's an issue and a really old spike at https://github.com/alphagov/govuk-frontend/issues/1139 – but didn't really get anywhere with it 😢